src/v/config/configuration.cc

```diff
@@ -1573,6 +1573,12 @@ configuration::configuration()
       {model::cloud_storage_chunk_eviction_strategy::eager,
        model::cloud_storage_chunk_eviction_strategy::capped,
        model::cloud_storage_chunk_eviction_strategy::predictive})
+  , cloud_storage_chunk_prefetch(
```

Comment:
nit: perhaps we could bound this property between between `[0, segment_size / chunk_size]`? The lower limit (0) can be enforced via `bounded_property`, but the upper limit is dynamic and has to be done somewhere else.

Reply:
I'll try to address this in a future PR